nordic/trezor/README.md

```diff
@@ -0,0 +1,77 @@
+# Trezor BLE Gateway
+
+Welcome to the **Trezor BLE Gateway** project!
+This repository contains the source code and instructions to build and flash the application onto the `t3w1_nrf52833` board.
+
+## Table of Contents
+
+- [Prerequisites](#prerequisites)
+- [Getting Started](#getting-started)
+    - [Install the toolchain](#install-the-toolchain)
+    - [Launch the nRF Shell](#launch-the-nrf-shell)
+    - [Initialize the Workspace](#initialize-the-workspace)
+    - [Update nRF Connect SDK Modules](#update-nrf-connect-sdk-modules)
+    - [Build the Application](#build-the-application)
+    - [Flash the Application](#flash-the-application)
+- [Contributing](#contributing)
+- [License](#license)
+
+## Prerequisites
+
+Before you begin, ensure you have met the following requirements:
+
+- **nrfutil**: Install [nrfutil](https://docs.nordicsemi.com/bundle/nrfutil/page/README.html). This tool is essential for managing the nRF Connect SDK and toolchains.
+- **Git**: Ensure you have Git installed for cloning repositories.
+
+## Getting Started
+
+Follow these steps to set up the project on your local machine.
+
+### Install the toolchain
+
+Using nrfutil, install the required toolchain for the nRF Connect SDK:
+```sh
+nrfutil toolchain-manager install --ncs-version v2.6.2
+```
+
+### Launch the nRF Shell
+
+First, launch the nRF shell using the `nrfutil` toolchain manager:
+
+```sh
+nrfutil toolchain-manager launch --shell
+```
+
+### Initialize the Workspace
+Initialize your West workspace for the Trezor BLE Gateway project:
+```sh
+cd west
+west init -l ./trezor
+```
+
+### Update nRF Connect SDK Modules
+
+Update the modules:
+```sh
+west update
```

Comment:
Can't yet find a way to have common SDK source so that it's not repeated by each repo clone (since now whole repo build is doubled to some 8 GB). Theoretically only build dir needs to be separate per project, but by  default they clobber over each other.

Reply:
we probably could put the west.yml into firmware repo root. then it would initialize the SDK outside of the firmware repo directory, and you could share it, but potential differences between west.yml might cause headaches. 

But you get the additional SDK bloat only per clone when you actually `west init` and `west update`, which might not be that often